test/service/test_response.py

```diff
@@ -1205,6 +1205,9 @@ def test_filter_by_projectId(self):
                                 self.assertEqual(test_data['title'], project['projectTitle'])
                             else:
                                 self.assertIn(test_data['title'], project['projectTitle'])
+                    for term in response_json['termFacets']['project']['terms']:
+                        self.assertIsInstance(term['projectId'], list)
+                        self.assertEqual(one(term['projectId']), test_data['id'])
```

Comment:
```suggestion
                        self.assertEqual(term['projectId'], [test_data['id']])
```

Reply:
Good idea, testing the singleton list value like this (instead of using one()) makes it unnecessary to first test if the value is a list.